### ISSUE TITLE
create-snowpack-app CLI fails 

### ISSUE THREAD

**Issue by yannbertrand** [pikapkg/create-snowpack-app#4]
I've tried the Vue, blank and React templates, but they all cause the same error:

```
  - Using template @snowpack/app-template-react
  - Creating a new project in /Users/yann/development/snowpack2/new-dir
(node:32733) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, stat '/Users/yann/development/snowpack2/new-dir/node_modules/@snowpack/app-template-react'
(Use `node --trace-warnings ...` to show where the warning was created)
(node:32733) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:32733) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Env: macOS Catalina, node@14/12/10

**Comment by FredKSchott** [comment]
fixed with `0f4deae`, thanks for reporting and sorry for the issue!

**Comment by yannbertrand** [comment]
Thank you for the quick fix!
